.github/CONTRIBUTING.md

```diff
@@ -1,40 +1,45 @@
-3. Please __do not__ submit PR's via Github's web interface. You should have a working [development envirorment](https://wiki.alpinelinux.org/wiki/Developer_Documentation#Development) available and submit your commits from the git interface.
+3. Please __do not__ submit PR's via Github's web interface. You should have a working [development envirorment](https://wiki.alpinelinux.org/wiki/Developer_Documentation#Development) available and submit your commits from your git client.
 
 ### Creating a Pull Request (PR)
 
-1. [Fork](https://help.github.com/articles/fork-a-repo/) our alpinelinux aports repository.
-2. Clone your copy of aports `git clone https://github.com/username/aports`
-3. Create a feature branch `git checkout -b my_new_feature`
-4. Make your desired changes and commit them with a correct commit message.
-  * Begin the commit message with a single short (less than 78 character) line summarizing the changes, followed by a blank line and then a more thorough description. Examples of correct formated summary lines are:
+1. [Fork](https://help.github.com/articles/fork-a-repo/) our Alpine Linux aports repository.
+2. Clone your copy of aports `git clone https://github.com/username/aports`.
+3. Create a feature branch `git checkout -b my_new_feature`.
+4. Make your desired changes and commit them with a [correct commit message](https://git-scm.com/book/ch5-2.html#Commit-Guidelines).
+  * Begin the commit message with a single short line summarizing the changes, followed by a blank line and then a more thorough description. Examples of correctly formated summary lines are:
     * When adding a new aport: testing/apkname: new aport
     * When modifying an aport: testing/apkname: short description about changes
   * If needed provide a proper formatted (line wrapped) description of what your patch will do. You can provide a description in the PR, but you must include a message for this specific commit in the commit description. If in the future we would like to distance ourselfs from Github the PR information could be lost.
-5. Open the aports repository in your github.com and switch to your feature branch. You should now see an option to create your PR. [More info](https://help.github.com/articles/creating-a-pull-request/)
-6. Wait for an Alpine Linux developer to review your changes
+5. Open your copy of the aports repository at github.com and switch to your feature branch. You should now see an option to create your PR. [More info](https://help.github.com/articles/creating-a-pull-request/)
+6. Wait for an Alpine Linux developer to review your changes.
 7. If all is ok your PR will be merged but if a developer asks for changes please do as followed.
   * Make the requested changes.
-  * add your file to git and `git commit --amend` (this keeps the commit log clean).
+  * Add your file(s) to git and commit (we will squash your commits if needed).
+  * Push your changes `git push --force origin my_new_feature`.
```

Comment:
`git push --force` is needed _only_ when you need to rewrite history of the branch. It must be used with caution, when you really know what you’re doing. It’s harmful to recommend it as default way to git beginners. One of the reasons for not using `--amend` or rebase by contributors is to avoid mistakes e.g. when the contributor accidentally push from wrong branch etc. The main reason is to keep history of the PR easier to track when doing review, i.e. avoid rewriting it.